### PR TITLE
updated tenant login

### PR DESCRIPTION
**Pull request by abilpraju-aot**
# Issue Tracking
https://aottech.atlassian.net/browse/FWF-1978
image

### CODE REVIEW

**Comment by sumesh-aot** [comment]
Or can we move the unauthorized route ?

File: forms-flow-web/src/components/PrivateRoute.jsx [hunk 1]
```diff
@@ -121,9 +122,18 @@ const PrivateRoute = React.memo((props) => {
...
         authenticate(kcInstance, props.store);
       } else {
-        instance.initKeycloak(() => {
-          authenticate(instance, props.store);
-          publish("FF_AUTH", instance);
+        instance.initKeycloak((authenticated) => {
+          if(!authenticated)
+          {
+           toast.error("The user does not belong to this tenant.",{autoClose: 3000}); 
+           setTimeout(function() {
```
Comment (shuhaib-aot) [inline forms-flow-web/src/components/PrivateRoute.jsx:129]: need to create an unauthorized route and redirect to it. i think that's better

**Review by shuhaib-aot — CHANGES_REQUESTED** [review]

File: forms-flow-web/src/components/PrivateRoute.jsx (hunk 1 above)
Comment (abhilash-aot) [reply forms-flow-web/src/components/PrivateRoute.jsx:129]: The user need to send back to Login Page for updating the User creds. This flow is fine for now. THis kind of scenarios will be rare though from one tenant user Trying to login to another tenant . But similarly we need to check for if a normal keycloak user tries to login withour a designer/ reviewer / client role should the message we different say tenant key in message wont be needed there. @abilpraju-aot

**Review by abhilash-aot — APPROVED** [review]

File: forms-flow-web/src/components/PrivateRoute.jsx (hunk 1 above)
Comment (shuhaib-aot) [reply forms-flow-web/src/components/PrivateRoute.jsx:129]: okay

**Review by shuhaib-aot — APPROVED** [review]

**Pull request merged**